string-manipulation-2/src/swap-chars.ts

```diff
@@ -0,0 +1,22 @@
+/* exported swapChars */
+
+function swapChars(
```

Comment:
You're using advanced syntax we haven't covered. There's a simpler solution: a simple for loop to go over each index of input, and using if/else conditionals to check we're looking at an index which needs to be swapped.